test/e2e/integreatly_test.go

```diff
@@ -94,16 +95,17 @@ func integreatlyWorkshopTest(t *testing.T, f *framework.Framework, ctx *framewor
 	}
 	//Product Stage - verify operators deploy
 	products := map[string]string{
-		"3scale":               "3scale-operator",
-		"amq-online":           "enmasse-operator",
-		"amq-streams":          "amq-streams-cluster-operator",
-		"codeready-workspaces": "codeready-operator",
-		"fuse":                 "syndesis-operator",
-		"launcher":             "launcher-operator",
-		"nexus":                "nexus-operator",
-		"solution-explorer":    "tutorial-web-app-operator",
-		"user-sso":             "keycloak-operator",
-		"ups":                  "unifiedpush-operator",
+		"3scale":                "3scale-operator",
+		"amq-online":            "enmasse-operator",
+		"amq-streams":           "amq-streams-cluster-operator",
+		"codeready-workspaces":  "codeready-operator",
+		"fuse":                  "syndesis-operator",
+		"launcher":              "launcher-operator",
+		"middleware-monitoring": "application-monitoring-operator",
```

Comment:
:+1:

Reply:
I've not merged this yet: https://github.com/integr8ly/manifests/pull/64 so the current e2e test run might fail I'd guess? Want to wait until this is merged, then rerun before we merge this one @mikenairn ?

Reply:
I would expect this to fail if the manifest PR isn't merged since the application registry is updated automatically on merge back to master, but i see it is actually here already https://quay.io/application/integreatly/integreatly-monitoring?tab=settings. Did you do this manually?

Reply:
Yes